helpers.go

```diff
@@ -51,6 +51,16 @@ func isTrue(obj interface{}) bool {
 	return false
 }
 
+func toSliceOfNumbers(values interface{}) []float64 {
+	_values := values.([]interface{})
```

Comment:
Should we add safe type assertion here to avoid panic?

Reply:
I don't think it's necessary because values must always be a slice and this is not a public method, so a panic here should be a good thing to those hacking this library. wdyt?

Reply:
You're right, it's already checked by `parseValues(values, data)` in `apply` function.

LGTM